android/src/main/java/io/bluedot/AppGeoTriggerReceiver.java

```diff
@@ -87,7 +94,18 @@ public void onZoneEntryEvent(@NotNull ZoneEntryEvent entryEvent, @NotNull Contex
         writableMap.putMap("zoneInfo", zoneDetails);
         writableMap.putMap("locationInfo", locationDetails);
         writableMap.putBoolean("isExitEnabled", entryEvent.isExitEnabled());
-
+        //Added a wait for 1 sec for 1st trigger to give App listeners time to register for callback
+        if (firstTrigger) {
+            Log.d("Plugin","Wait for 1 secs for first Entry");
```

Comment:
[I] Could we please improve the code formatting by adding spaces `"Plugin", "Wait`